src/components/CellComponent.js

```diff
@@ -12,11 +12,12 @@ export default class CellComponent extends Component {
     }
 
     render() {
-        const {passable, type, terrain} = this.props;
+        const {type, terrain, fogVisibility} = this.props;
         const cellClasses = classNames(
-            'cell', passable ? 'passable' : 'impassable', 
-            {player: type === 'player', end: type === 'end'}, 
-            {mountain: terrain === Cell.Terrain.MOUNTAIN && type === Cell.Terrain.NORMAL && passable}
+            'cell', terrain !== 'wall' ? 'passable' : 'impassable', 
+            {player: type === 'player', end: type === 'end', 
```

Comment:
style this obj better (one line per property)

Reply:
🐇